### PR TITLE
Add simanneal

### PR DESCRIPTION
**Pull request by tarcisiofischer**
The LICENSE file is just necessary temporarily, it was already created a PR to add a MANIFEST.IN
[Link for the PR](https://github.com/perrygeo/simanneal/pull/24)

### CODE REVIEW

File: simanneal/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0657354bacd503ff5fc0b984110be7b174ecedbb976caf1c230c73a128a06d5a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
```
Comment (beckermr) [inline simanneal/meta.yaml]: I believe this recipe can add the `noarch: python` directive to the build section.

File: simanneal/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  imports:
+    - simanneal
+
+about:
+  home: https://github.com/perrygeo/simanneal
+  license: ISC
+  license_family: BSD
+  license_file: LICENSE.txt
```
Comment (scopatz) [inline simanneal/meta.yaml]: The license file seems present in both the recipe here and the tarball.  can you remove the recipe here?

Comment (tarcisiofischer) [reply simanneal/meta.yaml]: @scopatz

**Comment by williamjamir** [comment]
> @williamjamir, are you OK with being a maintainer?

Yes

File: simanneal/meta.yaml (hunk 2 above)
Comment (scopatz) [reply simanneal/meta.yaml]: I didn't see it in the tarball. By saying `license_file: LICENSE.txt`, conda-build will look for it in the tarball.

Comment (igortg) [reply simanneal/meta.yaml]: removed

File: simanneal/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    - python
+
+test:
+  imports:
+    - simanneal
+
+about:
+  home: https://github.com/perrygeo/simanneal
+  license: ISC
+  license_family: BSD
```
Comment (scopatz) [inline simanneal/meta.yaml]: You need to have a license file entry as seen here: https://conda-forge.org/docs/meta.html#packaging-the-license-manually

File: simanneal/meta.yaml (hunk 2 above)
Comment (tarcisiofischer) [reply simanneal/meta.yaml]: Thanks, @igortg

**Comment by tarcisiofischer** [comment]
@scopatz Check again, please

**Comment by scopatz** [comment]
These files should be in the recipes directory

**Comment by scopatz** [comment]
Or rather in recipes/simanneal

**Pull request merged**